web/server/codechecker_server/api/report_server.py

```diff
@@ -662,6 +676,34 @@ def extended_data_db_to_api(erd):
         fileId=erd.file_id)
 
 
+def comment_data_db_to_api(comm):
+    return ttypes.CommentData(
+        id=comm.id,
+        author=comm.author,
+        message=formatComments(comm),
+        createdAt=str(comm.created_at),
+        kind=comm.kind
+    )
+
+
+def formatComments(comment):
```

Comment:
We are using snake_case function names in python code and I would also rename this function:
```suggestion
def get_comment_msg(comment):
```
Also please add a docstring to this function what this function does.